go2ts.go

```diff
@@ -422,11 +422,11 @@ func (g *Go2TS) populateInterfaceDeclarationProperties(interfaceDeclaration *typ
 	}
 }
 
-func (g *Go2TS) reflectTypeToTypeScriptType(reflectType reflect.Type, namespace string, wasExplicitlyAdded, ignoreNil bool) typescript.Type {
+func (g *Go2TS) reflectTypeToTypeScriptType(reflectType reflect.Type, namespace string, ignoreNil, wasExplicitlyAdded bool) typescript.Type {
```

Comment:
Here, and in the other places, change these values from 'bool' to an enum with two values, which will make all the call-sites easier to reason about, instead of having to add the inline comments, ala:

    ..., false /* =ignoreNil */)

Reply:
Done.